src/Commands/EnvCreateCommand.php

```diff
@@ -42,6 +44,8 @@ public function createBuildEnv(
             'message' => '',
             'pr-id' =>  '',
             'no-git-force' =>  false,
+            'create-before-push' => false,
+            'git-commit-compare' => 'HEAD',
```

Comment:
Hi @davereid-pfg looks great! Just one small request, could you please rename this option to `git-ref-compare`. I will go ahead and merge after that

Thanks!

Reply:
Changed!